lib/dialog-renderer/style.scss

```diff
@@ -0,0 +1,23 @@
+.dialog-renderer--overlay {
```

Comment:
I think we usually use underscores to signify a structure difference in sass?

`.dialog-renderer__overlay`

Reply:
Brain fart!! 🙈 Thank you 🙏🙏🙏